HelpSource/Guides/Tour-of-Special-Functions.schelp

```diff
@@ -600,12 +600,12 @@ var plotSpecs = [
 		"cylBesselIPrime(v, x)\nDerivative of the Modified Bessel Function of the First Kind \nvs=",
 		-25, 25
 	],
-	[	\cylBesselKPrime,
+	[   \cylBesselKPrime,
```

Comment:
for future reference - I think this is the style we prefer: https://github.com/supercollider/supercollider/wiki/Code-style-guidelines#needs-discussion-rule-place-each-element-of-a-multi-line-array-on-its-own-line

But we haven't made it an official rule yet (needs discussion)

Reply:
Thanks for noting it, Brian.  I typically follow the preferred style and in this exceptional case of it being rendered in the help page  I opted to condense it a bit — the examples, which are only supposed to be run as a block were getting really long. In hindsight I see it could've been done differently and would opt to stick with the preferred style in the future.